frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -58,15 +60,22 @@ export const monitoringAlertColumn: MonitoringAlertColumn[] = [
     fieldName: 'alertState',
     sortFunc: 'alertingRuleStateOrder',
   },
+  { title: 'Notifications', transforms: [cellWidth(20)] },
   { title: '' },
 ];
 
-export const monitoringAlertRows = (alertrules: Rule[], collapsedRowsIds: string[]) => {
+export const monitoringAlertRows = (
+  alertrules: Rule[],
+  collapsedRowsIds: string[],
+  namespace: string,
+) => {
   const rows = [];
   _.forEach(alertrules, (rls) => {
     rows.push({
       ...(rls.state !== RuleStates.Inactive && {
-        isOpen: rls.state === RuleStates.Firing && !_.includes(collapsedRowsIds, rls.name),
+        isOpen:
+          (rls.state === RuleStates.Firing && !_.includes(collapsedRowsIds, rls.name)) ||
+          (rls.state !== RuleStates.Firing && _.includes(collapsedRowsIds, rls.name)),
```

Comment:
Is this right? Shouldn't collapsed rows always be `isOpen = false`?

Reply:
yes, collapsed rows always be `isOpen = false`. But when the user expands it should be expanded to handle this scenario I added this. I missed this scenario in my last PR.